### PR TITLE
Location visibility - updated

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | master, 2.5

**Section updated:** Content management - Location visibility

### CODE REVIEW

File: docs/guide/content_management.md [hunk 1]
```diff
@@ -71,20 +71,28 @@ Once a Content item is published, it cannot be un-published.
...
 
-All Locations are visible by default. If a Location is made invisible manually, its status is set to Hidden.
+All Locations and Content items are visible by default.
+If a Location is made invisible manually, its status is set to Hidden.
 All Locations under it will change status to Hidden by superior.
+A Content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
+
+In the following example, the Content item 1 is Hidden by superior in the Location A while still visible in the Location B.
+
+![Visibility in two locations](img/locations_visibility.png)
```
Comment (MagdalenaZuba) [inline docs/guide/content_management.md]: I would change it to a diagram to keep it consistent throughout the page. Screenshot can go to User doc where we update screens after every release.

Comment (michalmilc90) [reply docs/guide/content_management.md]: Diagram added. Proceeding with updates in the user doc. ;)

File: docs/guide/content_management.md [hunk 2]
```diff
@@ -71,20 +71,28 @@ Once a Content item is published, it cannot be un-published.
...
-- Hidden by superior
+- Hidden by superior
 
-All Locations are visible by default. If a Location is made invisible manually, its status is set to Hidden.
+All Locations and Content items are visible by default.
+If a Location is made invisible manually, its status is set to Hidden.
 All Locations under it will change status to Hidden by superior.
+A Content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
+
+In the following example, the Content item 1 is Hidden by superior in the Location A while still visible in the Location B.
```
Comment (MagdalenaZuba) [inline docs/guide/content_management.md]:
```suggestion
In the following example, the **Content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
```

File: docs/guide/content_management.md (hunk 1 above)
Comment (michalmilc90) [reply docs/guide/content_management.md]: After further investigation, I decided not to add anything to the User doc as it provides a very clear explanation.

**Pull request merged**